lib/pages/chat/chat.dart

```diff
@@ -622,21 +619,8 @@ class ChatController extends State<Chat>
   void _openDownloadedFileForPreview({
     required DownloadFileForPreviewResponse downloadFileForPreviewResponse,
   }) async {
-    final mimeType = downloadFileForPreviewResponse.mimeType;
-    final openResults = await OpenFile.open(
-      downloadFileForPreviewResponse.filePath,
-      type: mimeType,
-      uti: DocumentUti(SupportedPreviewFileTypes.iOSSupportedTypes[mimeType])
-          .value,
-    );
-    Logs().d(
-      'ChatController:_openDownloadedFileForPreview(): ${openResults.message}',
-    );
-
-    if (openResults.type != ResultType.done) {
-      await Share.shareXFiles([XFile(downloadFileForPreviewResponse.filePath)]);
-      return;
-    }
+    await Share.shareXFiles([XFile(downloadFileForPreviewResponse.filePath)]);
```

Comment:
i thought share sheet apply for android only, open files in iOS is fine

Reply:
@hoangdat wdyt

Reply:
try to open and preview what we can, if can not, export it to File explorer.